bundles/alpha.model/src/alpha/model/transformation/reduction/SimplifyingReductions.xtend

```diff
@@ -106,16 +108,21 @@ class SimplifyingReductions {
 		reuseDep.getAffs.map[aff | aff.getConstant]
 	}
 	
+	protected static def ReduceExpression createXadd(AbstractReduceExpression reduceExpr, BasicElements BE) {
```

Comment:
This is hoisted into its own function so that it can be called by RemoveEmbedding